packages/web-extension/src/walletManager/walletManager.types.ts

```diff
@@ -8,11 +11,15 @@ export interface WalletManagerProps {
 export interface WalletManagerActivateProps<P extends string | number = string, O = unknown> {
   /** User given name for the observableWallet being activated */
   observableWalletName: string;
```

Comment:
This should be replaced with name in metadata, that is stored in WalletRepository. We can add a constraint to metadata,e.g. `WalletManager<Metadata extends {name: string}>`
```suggestion
```

Reply:
done here https://github.com/input-output-hk/cardano-js-sdk/pull/1012/commits/813029a3123dac04ffb0f783c6e239a3c19ae8e9